doc/influxdb-v2-integration.md

```diff
@@ -0,0 +1,80 @@
+# InfluxDB v2 Integration
+
+The current plugin version uses InfluxDB v1.x authentication semantics (user name and password, and a database), while v2 is using APi token, bucket, and organization to regulate the acceess to the database.
+
+The InfluxDB Target configuration options deceptively indicate InfluDB v2 options, but it looks like the plugin still expects to connect to InfluxDB v1.x. See this ticket for more details https://issues.jenkins.io/browse/JENKINS-65830
```

Comment:
How is it deceptive? InfluxDB 2 allows authentication with username and password, even though token based authentication is the standard. Token based authentication is also on the roadmap, so I think an extensive documentation for a workaround is a massive overkill.

This plugin is also aimed to send data into a readily configured InfluxDB database. Instructions on how to set up the database is not really a job for this plugin. [Influxdata](https://docs.influxdata.com/influxdb/v2.0/install/) has pretty good instructions on how to properly configure InfluxDB.

Reply:
I was probably channeling my initial frustration with using the plugin – I thought it would be easy to configure and use it with InfluxDB v2, but I just couldn’t make the authentication work. The target configuration was “deceptively” simple 😊. 

Now, that I’ve reviewed the code I wouldn’t use the same wording but I’d focus on improving the documentation to define explicitly the supported type of authentication. It would help newbies like me to adopt influxdb_plugin faster. 

I did a brief review of the InfluxDB [influxdb-client-java](https://github.com/influxdata/influxdb-client-java/blob/2d5ca33ef678a8e6cc14bca6556ab2723fb54ce5/client/README.md) and I would say it relies strictly on tokens to authenticate clients.  

The version used by influxdb_client is old - 1.15.0. Although my initial focus was on SonarQube integration I’d be more than happy to update the client and implement the token-based authentication.  

In any case, the documentation was re-phrased and I tried to provide more context on the challenge and solution.

Reply:
Thank you, the documentation is much better with not-so-provocative wording.

If you want to add the token based authentication be my guest, but please create a separate PR for that. I was planning on adding the token authentication "soon" myself, but you're welcome to do it if you have motivation.

Reply:
I'd love to contribute to this project. Wiley is building a CI/CD Dashboard and both Jenkins and this plugin are foundational components for supplying the metrics for the dashboards.  

I’ll start with the Jira board for Jenkins for new ideas, and I’d definitely give a try to adding a Token authentication for InfluxDB v2.